src/wovnio/wovnphp/Diagnostics.php

```diff
@@ -120,12 +134,16 @@ private function initResults()
         );
     }
 
-    private static function getWovnIni()
+    private function getWovnIni()
     {
-        $settingFileName = isset($env['WOVN_CONFIG']) ? $env['WOVN_CONFIG'] : DIRNAME(__FILE__) . '/../../../../wovn.ini';
+        if (isset($this->env['WOVN_CONFIG'])) {
```

Comment:
I'm not very clear on how `WOVN_CONFIG` gets set, does this mean that it'll always use the json file instead of the ini if it's present? If so we should make sure to update the documentation.

Reply:
`WOVN_CONFIG` gets set by the upstream server, like in Apache and Nginx. This is passed into PHP as a kind of "global" variable. To set it, in Apache it will be in `.htaccess`, in Nginx it will be `fastcgi_params`

Reply:
Got it thanks. Looks good then.